package.json

```diff
@@ -9,7 +9,7 @@
     "build": "NODE_ENV=production tsc",
     "postbuild": "copyfiles -f -V src/views/* dist/app/views",
     "lint": "tslint -p .",
```

Comment:
I understand this hasn't been added as part of this work, but tslint has been [deprecated since 2019](https://palantir.github.io/tslint/) and [typescript-eslint](https://typescript-eslint.io/) is the preferred linting tool.